### ISSUE TITLE
BUG - Link to Guided Answer isn't clickable as the output is encoded

### ISSUE THREAD

**Issue by gregorwolf** [SAP/open-ux-tools#1897]
### Related Feature

Feature request: #1805

### Description

With the PR #1805 a link to the Guided Answer [How to validate ADT API services used for deployment configuration?](https://ga.support.sap.com/dtp/viewer/index.html#/tree/3046/actions/45995:45996:50742:46000:57266). But when an error occurs during deployment that provides a link to this Guided Answer it isn't helpful as the link can't be clicked do due it's encoding.

### Steps to Reproduce

1. Setup a project to be deployed to an ABAP System where the OData service /UI5/ABAP_REPOSITORY_SRV is not yet active
2. run `npm run deploy`
3. Confirm to deploy

### Expected results

```
error abap-deploy-task ZEXCEL_DU Change logging level to debug your issue
error abap-deploy-task ZEXCEL_DU        (see examples https://github.com/SAP/open-ux-tools/tree/main/packages/deploy-tooling#configuration-with-logging-enabled)
error builder:custom deploy-to-abap Request failed with status code 403. Guided Answers troubleshooting guide available at https://ga.support.sap.com/dtp/viewer/index.html#/tree/3046/actions/45995:45996:50742:46000:57266
error builder:custom deploy-to-abap Request failed with status code 403
```

### Actual results

```
error abap-deploy-task ZEXCEL_DU Change logging level to debug your issue
error abap-deploy-task ZEXCEL_DU        (see examples https://github.com/SAP/open-ux-tools/tree/main/packages/deploy-tooling#configuration-with-logging-enabled)
error builder:custom deploy-to-abap Request failed with status code 403. Guided Answers troubleshooting guide available at https:&#x2F;&#x2F;ga.support.sap.com&#x2F;dtp&#x2F;viewer&#x2F;index.html#&#x2F;tree&#x2F;3046&#x2F;actions&#x2F;57266
error builder:custom deploy-to-abap Request failed with status code 403
```

### Screenshots

![image](https://github.com/SAP/open-ux-tools/assets/2639535/973484a7-ae3f-42a0-a327-18919c56aa18)

#### Version/Components/Environment	
Add any other context about the problem here
OS:                      
- [X] Mac OS
- [ ] Windows
- [ ] Other

**Comment by tobiasqueck** [comment]
@ianquigley-sap

**Comment by tobiasqueck** [comment]
@devinea @ianquigley-sap my understanding is that the issue is fixed with #1915. Is the fix already consumed by the Fiori tools?

**Comment by devinea** [comment]
This seems to relate to tools-suite code. I create a corresponding bug 29814 there.

**Comment by devinea** [comment]
Fixed released in SAP Fiori Tools Version 1.13.5 .
